src/bin/tikv-ctl.rs

```diff
@@ -1082,7 +1082,45 @@ fn main() {
                  .short("f").long("file")
                  .takes_value(true)
                  .help("meta file path"))
-        );
+        )
+        .subcommand(
+            SubCommand::with_name("compact-whole-cluster")
+                .about("compact whole cluster in a column family in a specified range")
```

Comment:
compact the whole cluster

Reply:
Change the command name or modify the help info?